src/components/TeamForm/TeamForm.js

```diff
@@ -68,6 +97,30 @@ class TeamForm extends Component {
             onChange={this.handleChange('address')}
           />
         </FormGroup>
+        <FormGroup controlId="teamForm-vote-duration">
+          <ControlLabel>Sort duration</ControlLabel>
+          <OverlayTrigger trigger="focus" placement="right" overlay={popoverRight}>
+            <Button
+              bsSize="xsmall"
+              className="glyphicon glyphicon-question-sign"
+              style={{ margin: 5 }}
```

Comment:
If I'm going to create and import a CSS file, I should move this there as well, right?

Reply:
Yes.

Reply:
Ok, sounds good.  I'd like to keep some bottom margin on there, too, though.  Otherwise the button crowds the input below.

Reply:
Sure - interesting that `vertical-align` doesn't solve the problem, but whatever works, really.

Reply:
Eh, it looks okay with a `vertical-align` of `text-bottom` but still a little cramped, in my opinion.  Tell me what you think: 
![image](https://user-images.githubusercontent.com/1675993/34957783-049f0602-f9fd-11e7-8359-fbb5d7130b44.png)

Reply:
I think this looks fine. I want there to be the same amount of margin between the label text and the field that there is for other labels/fields, so having the button drop down a bit looks good to me.

Reply:
Okay, I'll go with this solution.